react/utils.js

```diff
@@ -181,3 +181,20 @@ export function findSlaWithChannel(item, channel) {
 
   return cheapestSla
 }
+
+export function removeAddressValidation(address) {
+  const newAddressEntries = Object.entries(address).map(([key, value]) => {
+    const newValue =
+      value.value == null
+        ? typeof value === 'object'
+          ? null
+          : value
+        : value.value
+
+    return [key, newValue]
+  })
+
+  const newAddress = Object.fromEntries(newAddressEntries)
```

Comment:
these methods are [polyfilled on checkout](https://github.com/vtex/vcs.checkout-ui/blob/master/src/index.html#L21), so we should be good 👀

Reply:
actually, only `Object.entries` is polyfilled... should we add `Object.fromEntries` as well?

Reply:
hmm nice catch! I think we should 🤔

Reply:
created here https://github.com/vtex/vcs.checkout-ui/pull/1322